src/pages/faq.tsx

```diff
@@ -0,0 +1,38 @@
+import CzegoSieSpodziewac from '@/assets/markdown/faq/czego-sie-spodziewac.md';
+import JakDolaczyc from '@/assets/markdown/faq/jak-dolaczyc.md';
+import JakieSaKorzysci from '@/assets/markdown/faq/jakie-korzysci.md';
+import JakieSaObowiazki from '@/assets/markdown/faq/jakie-sa-obowiazki.md';
+import KimJestescie from '@/assets/markdown/faq/kim-jestescie.md';
+import KtoMozeDolaczyc from '@/assets/markdown/faq/kto-moze-dolaczyc.md';
+import WJakichObszarach from '@/assets/markdown/faq/w-jakich-obszarach.md';
+import { Accordion, SectionWrapper } from '@/components';
+
+const itemsMarkdownPart1 = [
+  KimJestescie,
+  KtoMozeDolaczyc,
+  JakDolaczyc,
+  WJakichObszarach
+];
+
+const itemsMarkdownPart2 = [
+  JakieSaKorzysci,
+  CzegoSieSpodziewac,
+  JakieSaObowiazki
+];
```

Comment:
Could it be merged into one array?

Reply:
I couldn't find any other solution that would fully work. If it is merged into one, and I use solution proposed by Jędrzej it results in something like that:
![image](https://github.com/CodersCrew/coderscrew-website/assets/33060719/bb6dbb34-c635-436c-aa82-23e2b073ab80)